src/routes/PaymentCheckPage.tsx

```diff
@@ -394,6 +395,13 @@ export default function PaymentCheckPage() {
         itemSx={{ pl: 2, pr: 0, gap: 2 }}
         variant="body2"
       />
+
+      <PspPrivacyInfo
+        termsLink="https://www.pagopa.gov.it/it/prestatori-servizi-di-pagamento/elenco-PSP-attivi/"
+        privacyLink="https://www.pagopa.gov.it/it/prestatori-servizi-di-pagamento/elenco-PSP-attivi/"
+        pspName={pspSelected?.pspBusinessName}
+      />
+
```

Comment:
```suggestion

     {!!pspSelected && <PspPrivacyInfo
        termsLink="https://www.pagopa.gov.it/it/prestatori-servizi-di-pagamento/elenco-PSP-attivi/"
        privacyLink="https://www.pagopa.gov.it/it/prestatori-servizi-di-pagamento/elenco-PSP-attivi/"
        pspName={pspSelected.pspBusinessName}
      />}
```

This is the matching change regarding the one in the component.

Reply:
I've updated